### DISCUSSION TITLE
Are you still working on this program?

### DISCUSSION THREAD

**Discussion by NiceFeather** [Hazuki0x0/YuzuBrowser#202] [General]
Hi Hazuki, 

Are you still working on this program? The Yuzubrowser is a world-renowned and well-known Browser, and all of us know there are still some issues with the current program, do you have any upgrade plans or milestones in later?

Best regards,

**Comment by Avrution** [comment]
Hopefully work will continue.  

Would love a paid/donate version if fixes would actually be implemented quicker.

**Comment by ghost** [comment]
It's the most powerfull webview based browser and it would be a pitty if development had stopped.

**Comment by Toiletmason** [comment]
I agree. Would suck if development stopped.

**Comment by meDevil** [comment]
Someone who know about Kotlin can fork and develop the project.

**Comment by smaragdus** [comment]
I hope that Yuzu Browser is not abandoned for good as it is my favourite browser since the day I discovered it- I have used 9 Android browsers so far and none comes even close to Yuzu in terms of features and customization. I do not want to think that I may need to switch to other browser in the future.

**Comment by lawson85** [comment]
I hope to see further development

**Comment by Avrution** [comment]
At this point, I'd just be happy with the crash on opening closed tabs being fixed.  Almost everything else works pretty well.

**Comment by ghost** [comment]
Finally someone forked YuzuBrowser at github and development is ongoing as BerryBrowser!

https://play.google.com/store/apps/details?id=jp.ejimax.yuzubrowser&hl=en_US

**Comment by lawson85** [comment]
> Finally someone forked YuzuBrowser at github and development is ongoing as BerryBrowser!
> 
> https://play.google.com/store/apps/details?id=jp.ejimax.yuzubrowser&hl=en_US

It is berry good. Is there a Github page or it?, also Yuzu is compatible with my tablet but Berry is not. Android 6.0.1

**Comment by ghost** [comment]
> It is berry good. Is there a Github page or it?, also Yuzu is compatible with my tablet but Berry is not. Android 6.0.1

First impression is good. Seems to be more stable and has some useful settings!

 https://github.com/Ejimax

**Comment by ksuuk** [comment]
Seems it's based some very old Yuzu build, as it contains bugs, which were fixed in Yuzu two tears ago, also reader mode is missing. I could not import Yuzu bookmark and speed dial settings, however, settings file imported successfully, but actually without any changes made.

Wrote two messages to developer about issues and also asked possible github page location, no reply yet.

**Comment by lawson85** [comment]
Why don't they just keep going with Yuzu instead of making more browsers?

**Comment by ksuuk** [comment]
> Why don't they just keep going with Yuzu instead of making more browsers?

Greenify shows, that Berry also includes google cloud messaging and AdGuard also detected some strange connections, maybe it's needed for ad block, at the moment I disabled ad block and keep testing and checking  AdGuard logs.

And I'm also wondering, why someone, with coding skills, instead of contributing fixes, starts his own port/build. So  if  code isn't public and developer is silent, I guess, I'll skip Berry.

**Comment by ghost** [comment]
@ksuuk Thanks for investigating! According to exodus privacy report, there are two trackers namely google crashlytics (also included in Yuzu) and firebase analytics. The author states that in the privacy policy.

https://reports.exodus-privacy.eu.org/en/reports/jp.ejimax.yuzubrowser/latest/

https://github.com/Ejimax/YuzuBrowser/wiki/Privacy-Policy

But I agree, without source code here at github, we have to be cautious!

**Comment by ksuuk** [comment]
@didux70 

BTW, Yuzu doesn't use firebase and is not by Greenify detected as google cloud messaging using app. Also Berry doesn't ask permission to access storage and as this is built in permission, then such access is also impossible to deny.
Such behavior makes me a quite paranoid...

**Comment by ksuuk** [comment]
Finally got some answers from japanese developer, here is overview:

1) The settings file of Berry Browser has a different structure from Yuzu Browser. So, Berry cannot support Yuzu settings file.
2) Berry is based on Yuzu 5.1.0-beta04, and Berry had many changes. But however, at last for me, seems Berry still contains bugs, which were fixed in the latest Yuzu beta.
3) Reporting a bugs is possible via beta feedback on Google Play.
4) Reader mode was removed and will never be supported.
5) As stated in Berry privacy policy, analytics information is used to improve the Berry.
6) There are no plans to develop with open source.

So, let's wait for the next volunteer, with coding skills, willing to contain developing Yuzu Browser.

**Comment by smaragdus** [comment]
Great news, folks! [Yuzu Browser](https://github.com/hazuki0x0/YuzuBrowser) [version](https://github.com/hazuki0x0/YuzuBrowser/releases) [6.0.0](https://github.com/hazuki0x0/YuzuBrowser/releases/tag/v6.0.0) has been released several hours ago!

Many thanks to @hazuki0x0 for releasing a new long-awaited version! For me Yuzu Browser is the best browser for Android, better than Vivaldi, which is my second choice.
